content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1114,15 +1114,15 @@ used to determine if the user has applied settings different from the kubeadm de
 
 Used on: Node
 
-Label that kubeadm applies on the control plane nodes that it manages.
+Label that kubeadm applies on the control plane nodes that it manages. If this label set, [EndpointSlice controller](/docs/concepts/services-networking/topology-aware-routing/#implementation-control-plane) ignores that node.
```

Comment:
This is a specific detail and actually _lots_ of things pay attention to this label. We could list some of them. I think listing just one use might come over as misleading.

Reply:
Hi @sftim,
Thanks for the feedback, i have written below a few points as per my investigation on this label use case. can you review it.
 
It is a restricted label and kubeadm manually applies it using a privileged client after a node has been created. control-plane nodes are scaled automatically using this label:
      • Attaching this label to a node results in deploying control plane components on this node, connecting the new etcd node to the etcd cluster, and regenerating all the necessary certificates and config files.
      • Removing this label results in deleting all control plane components on a node, gracefully removing it from the etcd cluster, and regenerating all the necessary config files and certificates.

Please suggest,

Reply:
```suggestion
A marker label to indicate that the node is used to run
{{< glossary_tooltip text="control plane" term_id="control-plane" >}} components.
The kubeadm tool applies this label to the control plane nodes that it manages.
Other cluster management tools typically also set this taint.

You can label control plane nodes with this label to make it easier to schedule Pods only onto these
nodes, or to avoid running Pods on the control plane.
```

Reply:
> • Attaching this label to a node results in deploying control plane components on this node, connecting the new etcd node to the etcd cluster, and regenerating all the necessary certificates and config files.

No, that's not true.

> • Removing this label results in deleting all control plane components on a node, gracefully removing it from the etcd cluster, and regenerating all the necessary config files and certificates.

No, that's not true.

Sorry to be so strong here but Kubernetes doesn't work how you're stating it does. These labels are managed by cluster lifecycle management tools such as `kubeadm`. Adding this label to a node doesn't set it up as part of your control plane.